packages/server/src/server/handlers/telemetry.ts

```diff
@@ -43,15 +43,15 @@ export async function getTelemetryHandler({ mastra, body }: TelemetryContext) {
         )
       : undefined;
 
-    const traces = await storage.getTraces({
+    const traces = await storage.__getTraces({
       name,
       scope,
       page: Number(page ?? 0),
       perPage: Number(perPage ?? 100),
       attributes,
     });
 
-    return { traces };
+    return traces;
```

Comment:
This issue fixes something I noticed in playground ui where traces were inside a traces object.